plugins/skip-button-plugin/README.md

```diff
@@ -0,0 +1,88 @@
+# Pillarbox Web: SkipButtonPlugin
+
+> Introduce your plugin with a clear overview of its goals, providing
+> essential context and addressing fundamental questions.
+
+## Requirements
+
+To use this component, you need the following installed on your system:
+
+- Node.js
+
+## Quick Start
+
+To get started with this plugin, follow these steps:
+
+Add the `@srgssr` registry to your `.npmrc` file:
+
+```plaintext
+//npm.pkg.github.com/:_authToken=TOKEN
+@srgssr:registry=https://npm.pkg.github.com
+```
+
+Generate an authentication token by following this
+guide: [Authenticating with a personal access token][generate-token]
+
+You can now install it through `npm` the following command:
+
+```bash
+npm install --save @srgssr/pillarbox-web @srgssr/skip-button-plugin
+```
+
+For instructions on setting up Pillarbox, see the [Quick Start guide](SRGSSR/pillarbox-web#quick-start).
+
+Once the player is installed you can activate the plugin as follows:
+
+```javascript
+import Pillarbox from '@srgssr/pillarbox-web';
+import '@srgssr/skip-button-plugin';
+
+const player = new Pillarbox('my-player', { plugins: { 'skip-button-plugin': true } });
```

Comment:
skipButtonPlugin ?

Reply:
Thanks, it was in the template as `kebabCase` I changed it there too to `camelCase`.